src/darker/tests/conftest.py

```diff
@@ -32,7 +33,10 @@ def __init__(self, root: Path):
     @classmethod
     def create_repository(cls, root: Path) -> "GitRepoFixture":
         """Fixture method for creating a Git repository in the given directory"""
-        check_call(["git", "init"], cwd=root)
+        env = os.environ.copy()
+        # for testsing, ignore ~/.gitconfig settings like templateDir and defaultBranch
+        env['HOME'] = root
```

Comment:
```suggestion
        env["HOME"] = root
```

Darker's own test pipeline fails if non-Black-compliant code is committed on modified lines. And we don't the `--skip-string-normalization` option of Black anymore.

Reply:
How embarrassing that I didn't run darker before submitting a pr to darker  :D 

fixed, thanks!